automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -26,171 +28,460 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
+            withWorkspace(projectName, "AuthDomainSpec_create", Set(groupOneName, groupTwoName)) { workspaceName =>
+              withCleanUp {
+                val workspaceListPage = signIn(Config.Users.fred)
+
+                workspaceListPage.open.filter(workspaceName)
+                workspaceListPage.ui.looksRestricted(projectName, workspaceName) shouldEqual true
+              }
             }
           }
         }
       }
-    }
+      "contains the list of auth domain groups in the workspace summary page" in withWebDriver { implicit driver =>
+        withGroup("AuthDomainSpec") { groupOneName =>
+          withGroup("AuthDomainSpec") { groupTwoName =>
+            withCleanUp {
+              val workspaceListPage = signIn(Config.Users.fred)
 
-    "cannot lose its authorization domain when cloned" in withWebDriver { implicit driver =>
-      withGroup("AuthDomainSpec", List(Config.Users.george.email)) { authDomainName =>
-        withWorkspace(projectName, "AuthDomainSpec_share", Some(authDomainName)) { workspaceName =>
-          withCleanUp {
-            api.workspaces.updateAcl(projectName, workspaceName,
-              Config.Users.george.email, WorkspaceAccessLevel.Reader)
+              val workspaceName = "AuthDomainSpec_create_" + randomUuid
+              register cleanUp api.workspaces.delete(projectName, workspaceName)
+              val workspaceSummaryPage = workspaceListPage.createWorkspace(projectName, workspaceName, Set(groupOneName, groupTwoName)).awaitLoaded()
 
-            val listPage = signIn(Config.Users.george)
-            val summaryPage = listPage.openWorkspaceDetails(projectName, workspaceName)
+              workspaceSummaryPage.ui.readAuthDomainGroups should include(groupOneName)
+              workspaceSummaryPage.ui.readAuthDomainGroups should include(groupTwoName)
+            }
+          }
+        }
+      }
 
-            val cloneWorkspaceName = workspaceName + "_clone"
-            val cloneModal = summaryPage.ui.clickCloneButton()
-            cloneModal.ui.readPresetAuthDomain() should be(Some(authDomainName))
+      "when the user is in none of the groups" - {
+        "when shared with them" - {
+          "can be seen but is not accessible" in withWebDriver { implicit driver =>
+            withGroup("AuthDomainSpec") { groupOneName =>
+              withGroup("AuthDomainSpec") { groupTwoName =>
+                withWorkspace(projectName, "AuthDomainSpec_reject", Set(groupOneName, groupTwoName), List(AclEntry(Config.Users.ron.email, WorkspaceAccessLevel.Reader))) { workspaceName =>
+                  val workspaceListPage = signIn(Config.Users.ron)
+                  workspaceListPage.filter(workspaceName)
+
+                  workspaceListPage.ui.clickWorkspaceInList(projectName, workspaceName)
+                  // micro-sleep just long enough to let the app navigate elsewhere if it's going to, which it shouldn't in this case
+                  Thread sleep 500
```

Comment:
Should be able to replace this sleep with `workspaceListPage.ui.showsRequestAccessModal shouldEqual true`.